src/client/commands/nowplaying.js

```diff
@@ -15,7 +15,11 @@ module.exports = {
             const completed = (radio.playTime);
 
             message.nowplayingDescription = client.messages.nowplayingDescription.replace("%radio.station.name%", radio.station.name);
-            message.nowplayingDescription = message.nowplayingDescription.replace("%radio.station.owner%", radio.station.owner);
+            if(radio.station.name != radio.station.owner){
```

Comment:
no if needed you can do `radio.station.name != radio.station.owner ? radio.station.owner : ""`

Reply:
Now its updated.